google-map-directions.html

```diff
@@ -83,6 +91,24 @@
       travelMode: 'DRIVING',
 
       /**
+       * Options for rendering.
+       *
+       * @attribute rendererOptions
+       * @type object
+       * @default {}
+       */
+      rendererOptions: {},
```

Comment:
objects needs to be init in `created()`

https://www.polymer-project.org/0.5/docs/polymer/polymer.html#attrhinting